### DISCUSSION TITLE
Welcome to kiwisR Discussions!

### DISCUSSION THREAD

**Discussion by rywhale** [rywhale/kiwisR#18] [General]
## 👋 Welcome!
This is the place for asking questions and making suggestions. Remember to stay civil!

**Comment by WQ-KRCA** [comment]
Hey rywhale, thanks for this amazing package! I was wondering if you had a list of hubs that this package can connect to? Or if this package is limited to just the three. TIA

**Comment by rywhale** [comment]
Hey there, @WQ-KRCA  at the bottom of this repo's README page you'll see this 
![image](https://github.com/user-attachments/assets/ccc75278-8ba1-4b3c-808c-cb742f6a4fa5)

So you can easily connect to any publicly accessible KiWIS server by subbing in its url for the hub argument, e.g.

```{r}
library(kiwisR)

ki_station_list(hub = "https://your-kiwis-server.com/KiWIS/KiWIS?")
```

The three default options are publicly accessible servers that I was aware of when I initially wrote the package. I'd definitely be open to adding more if you're aware of others that are public😄
